tools/test-js-with-node

```diff
@@ -66,6 +92,43 @@ except OSError:
     print('Bad command: %s' % (command,))
     raise
 
+NODE_COVERAGE_PATH = 'var/node-coverage/coverage.json'
+
+if options.coverage and ret == 0:
+    try:
+        with open(NODE_COVERAGE_PATH, 'r') as f:
+            coverage_json = ujson.load(f)
```

Comment:
I would close the `try` block here, since then you don't end up with everything hugely indented for no reason.